test/actions/files.test.js

```diff
@@ -39,7 +39,7 @@ describe('Actions.Files', () => {
         const formBoundary = imageFormData.getBoundary();
 
         const fileUploadResp = await basicClient.
-            uploadFile(basicTeam.id, basicChannel.id, clientId, imageFormData, formBoundary);
+            uploadFile(basicTeam.id, basicChannel.id, imageFormData, formBoundary);
```

Comment:
I know we're testing the client driver with this but a whole action just got added that doesn't have any code coverage, can we add a unit test for the new action?

We also made changes to the create post action, can we update the unit test for that action to cover the new code?